src/components/Calculator.js

```diff
@@ -1,34 +1,67 @@
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
+import calculate from '../logic/calculate';
 
```

Comment:
- [ ] You don't need to import it again, but it's important to import it inside the `Calculator.js` in place of the `calculate.js` file, just like the example you have in my comment before, use the component to import is a better practice on React. Please work on it. 🐶